.github/workflows/windows.yml

```diff
@@ -1,10 +1,7 @@
 name: "Windows"
 
-on:
-  push:
-  pull_request:
-    branches:
-      - latest
+on: 
+  workflow_dispatch
```

Comment:
This file should also be restored to the original so that it isn't included in this change.